src/client/components/editCropImage/editCrop.style.css

```diff
@@ -0,0 +1,23 @@
+@import '/../theme.css';
+
+.crop {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin: 20px;
+  padding: 10px;
+}
+.edit-image {
+  border-radius: 50%;
+  border: 3px solid var(--light-grey);
+}
+.edit-icon {
+  background: var(--light-blue);
+  border-radius: 50%;
```

Comment:
I believe you're missing `border: none;` for it to match the design :)

Reply:
fixed